### PR TITLE
pfe-tools: added build ts files to js to build script

### PR DESCRIPTION
**Pull request by heyMP**
The `.js` files werent' being generated during build for pfe-tools causing the rest of the build commands to fail.

fixes: #1817

### CODE REVIEW

File: tools/pfe-tools/package.json [hunk 1]
```diff
@@ -41,12 +41,12 @@
...
   "scripts": {
-    "build": "tsc -b .",
+    "build": "node _scripts/build.js",
     "clean": "git clean -fx -e node_modules -e tsconfig.tsbuildinfo .",
-    "dev": "node _scripts/dev.js",
+    "dev": "node _scripts/build.js -- watch",
     "prepublishOnly": "run-s prepublishOnly:*",
     "prepublishOnly:analyze": "npm run analyze",
-    "prepublishOnly:build": "tsc -b ."
+    "prepublishOnly:build": "node _scripts/build.js"
```
Comment (bennypowers) [inline tools/pfe-tools/package.json:49]: can you run `prepublish` then `npm pack` and confirm the contents of the tarballs?

Comment (bennypowers) [reply tools/pfe-tools/package.json:49]: Also, this won't generate the .d.ts files, which we'd like in the tarball

Comment (heyMP) [reply tools/pfe-tools/package.json:49]: It's complaining about the `npm run analyze` script not being available.
Screen Shot 2021-12-14 at 8 40 00 AM

**Comment by heyMP** [comment]
This issue was solved by running `npm run clean:deep`.

**Pull request closed without merging**